### DISCUSSION TITLE
[Localization Problem] Incorrect Chinese Item Description

### DISCUSSION THREAD

**Discussion by Cybernetics9798** [FakeFishGames/Barotrauma#14391] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### **The maximum sustainable depths for various diving suits are terribly wrong in Chinese descriptions. This applies to both the traditional Chinese version and the simplified Chinese version.**

**The correct max depth for the combat diving suit should be 5700 m:**
![20240802152510_1](https://github.com/user-attachments/assets/9a2ef430-bcf3-45ab-a561-01127bdd449e)

But it's 6000 m in the Chinese version of the game, which unfortunately got my friend crushed by the water pressure the other day.
Simplified Chinese version:
![20240802152628_1](https://github.com/user-attachments/assets/7bc245a2-33b5-449f-b20a-5c112c654ed8)

Traditional Chinese Version:
![combat_TC](https://github.com/user-attachments/assets/25664aa1-d13b-4b6f-8127-db05c16c43be)

**The same goes for PUCS. It should be 6200 m but it's 6000 m in Chinese.**
English version:
![pucs_eng](https://github.com/user-attachments/assets/1e166bd3-3540-42ec-8a31-7517998ce2aa)

Simplified Chinese version:
![pucs_SC](https://github.com/user-attachments/assets/d20d9e3e-fd3f-4385-bb86-211ac05d6012)

Traditional Chinese version:
![PUCS_TC](https://github.com/user-attachments/assets/32b5121e-2679-4c76-8fdc-086c60eab28b)

**Also for the abyss diving suit (should be 6600 m but it's 10000 m in Chinese):**
English version:
![abyss_eng](https://github.com/user-attachments/assets/aa1a109f-e2f4-45d8-a722-260175aef43e)

Simplified Chinese Version:
![Abyss_SC](https://github.com/user-attachments/assets/54d49b14-2673-4212-bd90-76fc5a8f08bc)

Traditional Chinese version:
![abyss_TC](https://github.com/user-attachments/assets/0e9148da-6e54-488c-b9f2-da21d68e6dec)

**It's even worse for Slipsuit. The Chinese descriptions did not include a diving limit AT ALL.**
English version:
![slip_eng](https://github.com/user-attachments/assets/3827368e-5f68-4364-a099-9c7a4b7e0afd)

Simplified Chinese version:
![slip_SC](https://github.com/user-attachments/assets/da2b5f61-a01e-4c18-8071-6ba646011eb8)

Traditional Chinese version:
![slip_TC](https://github.com/user-attachments/assets/d67ceb88-4e5c-42c6-b9dd-e31b11b78b77)

### Some Other Potential Translation Improvements
I would recommend directly translating Naloxone as 纳洛酮 instead of 烯丙羟吗啡酮. The latter is much more confusing than the first one and is not a widely used translation in Chinese contexts. 纳洛酮 is the most common name for Naloxone.
![N](https://github.com/user-attachments/assets/21b4b2cc-ebd0-4306-8dbc-b98edfc9cc05)

The item "plastic seal" is translated as "elastic manmade skin" (可塑人造皮肤) in Chinese, which does not align with the original item name. If I were the person who would translate this, I prefer to put 塑料绷带 (plastic bandage) here instead of elastic manmade skin.
![seal](https://github.com/user-attachments/assets/fb904749-d2c3-4fad-9b81-7c81ebd0d92e)

Although the translation for Chloral Hydrate is correct (水合三氯乙醛), Chinese chemists usually call it 水合氯醛 in short.
![20240802172103_1](https://github.com/user-attachments/assets/81c1a9fe-2e5c-41e3-970f-345077c3a26c)

### Reproduction steps

1. Switch the game to the Chinese language pack
2. Dive under 5700 m with a combat diving suit
3. You'll find the item description is trolling

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Also in TurkishVanilla.xml entitydescription.divingsuit doesn't mention that you will pop below 4000m

**Comment by A2ur3** [comment]
other Chinese translation issues:
SimplifiedChineseVanilla.xml
line 1084 - <respawningin> 内复活
line 3640 - <entityname.op_researchterminal> 前哨站
line 4471 - <dialogtargetdown> 敌人来袭
line 4472 - <dialogtargetdown> 目标来袭
line 4455 - <dialogcannotreachpatient>
line 4815 - <orderdialog.prepareforexpedition> 前厅
line 8603 - <missionsuccess.lostcargo> 改回收
line 8604 - <missionfailure.lostcargo> 改回收
line 8612 - <missionsuccess.lostartifact> 改回收
line 8613 - <missionfailure.lostartifact> 改回收
line 8622 - <missionfailure.cargowreckthalamus1> 改回收
SimplifiedChineseVanillaEndgame.xml
line 28 - <missionsonarlabel.gotoend2.switch> 切换

**Comment by Regalis11** [comment]
Thank you for the report! We've reported the issues to the localization team, these should be fixed in the next update.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/8ef1017c1ce425ffb08c5d7137f1b826ebd725df (commit in our private repo), fix coming up in the next update.
